helm-chart/renku-gateway/templates/_helpers.tpl

```diff
@@ -0,0 +1,43 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "renku-gateway.name" -}}
```

Comment:
Use `gateway` to stay consistent with the other sub-charts ;)

```
define "gateway.name"
...
```

Reply:
Actually, I was following the ui example (see https://github.com/SwissDataScienceCenter/renku-ui/blob/master/helm-chart/renku-ui/templates/_helpers.tpl). Should I change it there too?

Reply:
yes, make an issue, but it's not urgent ;)